WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyControlTileViewModel.cs

```diff
@@ -27,7 +28,11 @@ public PrivacyControlTileViewModel(WalletViewModel walletVm, bool showPrivacyBar
 		_walletVm = walletVm;
 		_showPrivacyBar = showPrivacyBar;
 
-		ShowDetailsCommand = ReactiveCommand.Create(ShowDetails);
+		var showDetailsCanExecute =
+			walletVm.WhenAnyValue(x => x.IsEmptyWallet)
```

Comment:
The usage of `IsEmptyWallet` is not correct, the naming is misleading. If you receive some funds send them away it will give you `false` as the transaction history have some items.

You could use `IsWalletBalanceZero` instead. If that is true then the wallet has zero coins for sure.